pkg/sink/codec/encoder.go

```diff
@@ -83,3 +83,10 @@ func IsColumnValueEqual(preValue, updatedValue interface{}) bool {
 	// the value type should be the same
 	return preValue == updatedValue
 }
+
+// TopicPartitionKey contains the topic and partition key of the message.
+type TopicPartitionKey struct {
```

Comment:
It looks like this should not be a part of the `codec`

Reply:
Put it here can make the import graph happier, and actually, the encoderGroup in the codec package already relies on the topic and partition information.